homeworks/serhii.ripchanskyi_Ripchanskiy/6-typescript/src/index.ts

```diff
@@ -0,0 +1,79 @@
+import './styles.css';
+
+const scoreElement = document.querySelector('[data-score]') as HTMLHeadingElement;
+const cubeElement = document.querySelector('[data-cube]') as HTMLDivElement;
+const cubeScoreElement = document.querySelector('[data-cube-score]') as HTMLDivElement;
+const keyElement = document.querySelector('[data-key]') as HTMLDivElement;
+const alphabet: string = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+
+class Game {
+    private score: number = 100;
+    private currentKey: string = '';
+    private inputKey: string = '';
+    private interval: number = 2000;
+    private timerId: NodeJS.Timeout;
+
+    constructor(
+        private scoreElement: HTMLHeadingElement,
+        private cubeScoreElement: HTMLDivElement,
+        private keyElement: HTMLDivElement,
+        private cubeElement?: HTMLDivElement,
+    ) {}
+
+    start() {
+        scoreElement.innerText = String(this.score);
```

Comment:
Several points:
1) You should update the `scoreElement` from `this`, not from global variable. Simply use `this.scoreElement.innerText = ......`
2) Create a separate method `updateGameHeader(header: string | number)`, that sets the scoreElement.innerText.
Hint:
use `this.scoreElement.innerText = String(header);`